src/global/Id.h

```diff
@@ -16,6 +17,7 @@ using Id = ValueId;
 typedef uint16_t Score;
 
 // TODO<joka921> Make the following ID and index types strong.
+// BOOST_STRONG_TYPEDEF(uint64_t, ColumnIndex)
```

Comment:
Please d
```suggestion
```
In `global/IndexTypes.h` we role our own `strong typedef` class. It supports hashing, but using it here immediately would require many additional changes (mostly all the places where the `ColumnIndex`es are accessed would have to be changed to again extract the actual contained value. I would not suggest mixing this with the `making stuff work on Mac` effort.

Reply:
I see. The boost type was helpful since it mostly works without other changes, so having it in place temporarily, made it easier to implement this PR (until I hit the hashing issue) Anyhow, I'll just consider it a useful tool for the process, and leave the code as is for now. I agree we shouldn't mix that with this effort.

If we can eventually add a Mac CI build, that would also solve the problem of keeping the data types more consistent, without needing to use a strong type.